Src/Newtonsoft.Json/Converters/IsoDateTimeConverter.cs

```diff
@@ -124,7 +124,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
         public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
         {
             bool nullable = ReflectionUtils.IsNullableType(objectType);
-#if !NET20
+#if HAVE_NULLABLE
```

Comment:
Every build should have nullable. I'm confused why this even has an `#if` around it.